app/_scss/_theme_vars.scss

```diff
@@ -61,7 +61,7 @@ $Theme-font-sizes: (
         line-height: 1.86 // 26px / 14px
       ),
       large: (
-        font-size: 20px,
+        font-size: 18px,
```

Comment:
This will affect all the headings in the page. Are we good with that? @groupbuddies/designers